Extensions/PathfindingBehavior/tests/commonpathfindingruntimebehavior.spec.js

```diff
@@ -1,10 +1,11 @@
 // @ts-check
-describe('gdjs.PathfindingRuntimeBehavior', function () {
+describe.only('gdjs.PathfindingRuntimeBehavior', function () {
```

Comment:
I forgot an `only` here, it's still on the master. I'll do a PR.

Reply:
Thanks for noticing! We should probably create a linter rule or github action or something like this that warns against `.only`